webcomponents/tool/src/main/frontend/js/rubrics/sakai-rubric-criterion-student.js

```diff
@@ -64,7 +77,17 @@ export class SakaiRubricCriterionStudent extends RubricsElement {
                   <div class="rating-item student ${r.selected ? "selected" : ""}" id="rating-item-${r.id}">
                     <h5 class="criterion-item-title">${r.title}</h5>
                     <p>${r.description}</p>
-                    <span class="points">${r.points.toLocaleString(this.locale)} Points</span>
+                    <span class="points">
+                      ${this.weighted && r.points > 0 ?
+                        html`
```

Comment:
Move the html onto the same line as your ? ?

Reply:
Done, missed that one 😅